debian/compat

```diff
@@ -1 +1 @@
-9
+12
```

Comment:
You also should change the `debhelper` `Build-Depends` to ensure a new enough debhelper is installed. The proper way to do this is to set it to `debhelper-compat (=12)` per https://manpages.debian.org/unstable/debhelper/debhelper.7.en.html#COMPATIBILITY_LEVELS.

Reply:
Thanks ⇒ https://github.com/endlessm/eos-updater/pull/248